src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Streams/Given_DataWriter.cs

```diff
@@ -0,0 +1,786 @@
+﻿using System;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.UI.RuntimeTests.Extensions;
+using Windows.Storage.Streams;
+using UnicodeEncoding = Windows.Storage.Streams.UnicodeEncoding;
+
+namespace Uno.UI.Tests.Windows_Storage.Streams
+{
+	[TestClass]
+	public class When_DataWriter
+	{
+		[TestMethod]
+		public async Task When_WriteThenRead()
+		{
+			bool saveBool = true;
+			byte saveByte = 10;
+			int saveInt = 12345;
+			double saveDouble = 12345.678;
+			DateTimeOffset saveDate = DateTimeOffset.Now;
+			TimeSpan saveTimeSpan = TimeSpan.FromMinutes(10);
+			Guid saveGuid = Guid.NewGuid();
+
+			var writer = new DataWriter();
+			try
+			{
+				writer.WriteBoolean(saveBool);
+				writer.WriteByte(saveByte);
+				writer.WriteInt32(saveInt);
+				writer.WriteDouble(saveDouble);
+				writer.WriteDateTime(saveDate);
+				writer.WriteTimeSpan(saveTimeSpan);
+				writer.WriteGuid(saveGuid);
+			}
+			catch (Exception ex)
+			{
+				Assert.Fail($"Error writing with DataWriter - {ex.Message}");
+			}
+
+			var buffer = writer.DetachBuffer();
+
+			var reader = DataReader.FromBuffer(buffer);
+
+			// variables below has to be initialized, because if not, we get CS0165 Use of unassigned local variable
+			// Codacy shows false errors here, it should be ignored.
+			bool loadBool = false;
+			byte loadByte = 0;
+			int loadInt = 0;
+			double loadDouble = 0;
+			DateTimeOffset loadDate = new DateTimeOffset(2000, 1, 1, 0, 0, 0, TimeSpan.FromMinutes(10));
+			TimeSpan loadTimeSpan = TimeSpan.FromSeconds(10);   // and saved value was 10 minutes
+			Guid loadGuid = Guid.Empty;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'loadGuid'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)